src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php

```diff
@@ -61,7 +65,7 @@ public function register_routes() {
 		// Lists/updates a single global style variation based on the given id.
 		register_rest_route(
 			$this->namespace,
-			'/' . $this->rest_base . '/(?P<id>[\/\s%\w\.\(\)\[\]\@_\-]+)',
+			'/' . $this->rest_base . '/(?P<id>[\/\w-]+)',
```

Comment:
The change here reverts the changes from [changeset 52376](https://core.trac.wordpress.org/changeset/52376) to restore the original regex.